iidm/iidm-serde/src/test/java/com/powsybl/iidm/serde/AreaSerDeTest.java

```diff
@@ -0,0 +1,70 @@
+/**
+ * Copyright (c) 2024, Coreso SA (https://www.coreso.eu/) and TSCNET Services GmbH (https://www.tscnet.eu/)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ * SPDX-License-Identifier: MPL-2.0
+ */
+package com.powsybl.iidm.serde;
+
+import com.powsybl.commons.PowsyblException;
+import com.powsybl.iidm.network.*;
+import org.junit.jupiter.api.Test;
+
+import java.io.IOException;
+import java.io.InputStream;
+import java.time.ZonedDateTime;
+
+import static com.powsybl.iidm.serde.IidmSerDeConstants.CURRENT_IIDM_VERSION;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+/**
+ * @author Marine Guibert {@literal <marine.guibert at artelys.com>}
+ * @author Valentin Mouradian {@literal <valentin.mouradian at artelys.com>}
+ */
+class AreaSerDeTest extends AbstractIidmSerDeTest {
```

Comment:
There should be a SerDe test with subnetworks to check that it works as expected.